### ISSUE TITLE
GeoQueryUserHaversine returns 0 results when order argument is used

### ISSUE THREAD

**Issue by baden03** [birgire/geo-query#14]
Not sure if the document needs to be updated (see: https://github.com/birgire/geo-query/issues/7) or if there is a bug, but using the _order_ parameter returns zero results when using GeoQueryUserHaversine. Omitting the order parameter returns as expected.
```
'geo_query' => [
		'lat'       	       	=> $data['lat'],
		'lng'          	=> $data['lng'],
		'lat_meta_key'  => 'lat',
		'lng_meta_key' => 'lng',
		'radius'       	=> 10,
		'order'       	=> 'DESC',
		'context'       	=> '\\Birgir\\Geo\\GeoQueryUserHaversine',
	 ],
```

**Comment by baden03** [comment]
In the meantime, here is an expensive PHP 7+ workaround to sort the results by distance: (actually called distance_value):

```
$users = $user_query->get_results();
usort($users, function ($user1, $user2) {
    return $user1->distance_value <=> $user2->distance_value;
});
```

**Comment by birgire** [comment]
Thank you @baden03 and for the temporary fix.

I'm looking into it in PR #15 

So hopefully this will fix the order parameter within the geo query.

Then the idea is to handle #7 hopefully in the near future.

**Comment by birgire** [comment]
I close the issue for now, as this should be fixed in version 0.1.1

but please reopen if needed, 

thanks
